websites-css/project.css

```diff
@@ -2,20 +2,82 @@
     box-sizing: border-box;
 }
 
+body {
+    display: flex;
+    height: 100%;
+    flex-direction: column;
+}
+
+html {
+    height: 100vh;
+}
+
 .header {
     background-color: var(--color-header-bg);
+    margin: 0;
```

Comment:
You should not have this header class as i have already made a header, footer and its respective css is in the master.css file.

Reply:
i have removed it and updated